.travis.yml

```diff
@@ -5,14 +5,14 @@ services:
 
 before_install:
  - echo $PWD
- - docker pull dalg24/vera_tpls
+ - docker pull bcollins/futility_tpl
```

Comment:
Documentation for this docker image would be helpful.

Reply:
Yeah, I'm working on automating the process and then adding the script to the repo but its basically the vera_tpls docker with Futility+Trilinos built and installed.